### DISCUSSION TITLE
Completed issues

### DISCUSSION THREAD

**Discussion by Dafnik** [nartc/ngxtension-platform#466] [General]
Hello!
I wanted to do a little bit of repository maintenance and found following issues. (which can be closed in my opinion).

Completed issues:
- #126 new logo variant has been created and applied everywhere.
- #244 we now have a loading indicator utility.
- #293 this is done. (The linked pr is merged)

Not so sure if completed:
- #125 The ngxtension website now has the "What does ngxtension offer 🔨?" section. So perhaps it counts as done.

Storage Helper Duplicates:
- #245
- #42
`injectLocalStorage` was added and there is a third ticket for adding the docs. (#464)

Do with this information whatever you like.

**Comment by jdegand** [comment]
There are more that can be closed: #330, #85, #42, and #346.

**Comment by eneajaho** [comment]
@Dafnik Thank you very much for doing this 💙! 
Just closed all of them 😄🙌
